<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-10-23T12:44:06 IST" name="Default suite" finished-at="2025-10-23T12:45:17 IST" duration-ms="70563">
    <groups>
    </groups>
    <test started-at="2025-10-23T12:44:06 IST" name="Default test" finished-at="2025-10-23T12:45:17 IST" duration-ms="70563">
      <class name="newTestng.validlogin1">
        <test-method is-config="true" signature="setup()[pri:0, instance:newTestng.validlogin1@4efbca5a]" started-at="2025-10-23T12:44:07 IST" name="setup" finished-at="2025-10-23T12:44:22 IST" duration-ms="15590" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="invalidlogin()[pri:0, instance:newTestng.validlogin1@4efbca5a]" started-at="2025-10-23T12:44:22 IST" name="invalidlogin" finished-at="2025-10-23T12:44:48 IST" duration-ms="25945" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[did not expect [OrangeHRM] but found [OrangeHRM]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: did not expect [OrangeHRM] but found [OrangeHRM]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failEquals(Assert.java:1582)
at org.testng.Assert.assertNotEqualsImpl(Assert.java:157)
at org.testng.Assert.assertNotEquals(Assert.java:2201)
at org.testng.Assert.assertNotEquals(Assert.java:2223)
at newTestng.validlogin1.invalidlogin(validlogin1.java:45)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidlogin -->
        <test-method is-config="true" signature="quit()[pri:0, instance:newTestng.validlogin1@4efbca5a]" started-at="2025-10-23T12:44:48 IST" name="quit" finished-at="2025-10-23T12:44:49 IST" duration-ms="617" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
        <test-method is-config="true" signature="setup()[pri:0, instance:newTestng.validlogin1@4efbca5a]" started-at="2025-10-23T12:44:49 IST" name="setup" finished-at="2025-10-23T12:44:58 IST" duration-ms="9022" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="login()[pri:0, instance:newTestng.validlogin1@4efbca5a]" started-at="2025-10-23T12:44:58 IST" name="login" finished-at="2025-10-23T12:45:16 IST" duration-ms="18729" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="quit()[pri:0, instance:newTestng.validlogin1@4efbca5a]" started-at="2025-10-23T12:45:16 IST" name="quit" finished-at="2025-10-23T12:45:17 IST" duration-ms="577" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- newTestng.validlogin1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
